api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -187,11 +192,28 @@ public void validateInstitutionalAffiliation(Profile profile) {
    * @param previousProfile
    */
   public void updateProfileForUser(DbUser user, Profile updatedProfile, Profile previousProfile) {
+    updateProfileForUser(user, updatedProfile, previousProfile, false);
+  }
+
+  /**
+   * Updates a profile for a given user as Admin and persists all information to the database.
+   *
+   * @param user
+   * @param updatedProfile
+   * @param previousProfile
+   */
+  public void adminUpdateProfileForUser(
```

Comment:
Why do we need the "forUser" suffix? I think that was there previously but I never got what it meant.

So Admins can't create profiles, right? At least not in their capacity as admins.

Reply:
(NOTE: I have removed the admin version of this method in favor of checking authorities)

I'm not sure.  @gjuggler this was your originally, right?  Any concern with renaming `updateProfileForUser()` to "updateProfile" ?

Reply:
IIRC this method originally lived at the controller level, where there is also an updateProfile() method (which handles the case when "my current user" is the target user being updated). Now that it's in the service it makes sense to remove the suffix for consistency with other methods in that file.

Reply:
Thanks.  Updated.